util/generate-ami-list.py

```diff
@@ -36,23 +35,23 @@ def get_ami_list(regions, date, version):
 
     for region_name in regions:
         try:
-            conn = boto.ec2.connect_to_region(region_name)
-            images = conn.get_all_images(owners=owners, filters={"name": "cfncluster-" + version + "*" + date})
+            ec2 = boto3.client('ec2', region_name=region_name)
+            images = ec2.describe_images(Owners=owners, Filters=[{'Name': 'name', "Values": ["cfncluster-%s*%s" % (version, date)]}])
 
             amis = OrderedDict()
-            for image in images:
+            for image in images.get('Images'):
                 for key, value in distros.items():
-                    if value in image.name:
-                        amis[key] = image.id
+                    if value in image.get('Name'):
+                        amis[key] = image.get('ImageId')
 
             if len(amis) == 0:
-                raise SystemExit
+                print("Warning: there are no AMIs in the selected region (%s)" % region_name)
             else:
                 amis_json[region_name] = amis
 
         except SystemExit:
```

Comment:
You can remove this catch too, 
because you removed the `raise SystemExit` in the row 49, so the _SystemExit_ exception cannot occur anymore.

Reply:
removed.